include/quicly/loss.h

```diff
@@ -226,15 +226,13 @@ inline uint32_t quicly_rtt_get_pto(quicly_rtt_t *rtt, uint32_t max_ack_delay, ui
 inline void quicly_loss_init(quicly_loss_t *r, const quicly_loss_conf_t *conf, uint32_t initial_rtt, const uint16_t *max_ack_delay,
                              const uint8_t *ack_delay_exponent)
 {
-    *r = (quicly_loss_t){.conf = conf,
```

Comment:
Had to put this back to get compilation working again. I don't know why it can't use this style of initialization. These are the errors:

`C:\Users\sghiocel\Documents\sdwan_after_s1_merge\libsdwan\extern\public\quicly\include\quicly/sentmap.h(258,32): error C4576: a parenthesized type followed by an initializer list is a non-standard explicit
 type conversion syntax [C:\Users\sghiocel\Documents\sdwan_after_s1_merge\libsdwan\build\windows\Debug\src\sdwan.vcxproj]
C:\Users\sghiocel\Documents\sdwan_after_s1_merge\libsdwan\extern\public\quicly\include\quicly/loss.h(229,26): error C7555: use of designated initializers requires at least '/std:c++latest' [C:\Users\sghioc
el\Documents\sdwan_after_s1_merge\libsdwan\build\windows\Debug\src\sdwan.vcxproj]
C:\Users\sghiocel\Documents\sdwan_after_s1_merge\libsdwan\extern\public\quicly\include\quicly/loss.h(237,47): error C4576: a parenthesized type followed by an initializer list is a non-standard explicit ty
pe conversion syntax [C:\Users\sghiocel\Documents\sdwan_after_s1_merge\libsdwan\build\windows\Debug\src\sdwan.vcxproj]`

increasing to c++20 only fixes the error that asks for "requires at least '/std:c++latest'"

Reply:
So you are getting this error when building libsdwan (which includes `quicly.h`), not quicly library itself, right? This is because of c++ compiler. Ok, but let's use a more classical C-style approach then:

```
memset(r, 0, sizeof(*r));
r->conf = conf;
[...]
```

Reply:
I'll change this in `refact`.